packages/editor-ui/src/App.vue

```diff
@@ -214,7 +213,22 @@ export default mixins(
 
 .sidebar {
 	grid-area: sidebar;
+	height: 100vh;
 	z-index: 999;
 }
 </style>
 
+<style lang="scss">
+.root-container {
+	.centered-content {
+		padding: var(--spacing-3xl) var(--spacing-3xl) var(--spacing-4xl) 0;
+	}
+}
+
+.root-container.menu-collapsed {
+	.centered-content {
+		padding-left: var(--spacing-3xl);
+	}
+}
```

Comment:
My idea was that this will be the only place where this is defined and then we can just apply the class if we want to have a centered view. Any ideas on how to solve this differently?

Reply:
My goal was to have it in a global class so we can just apply it on content container that needs to be centered. Any ideas on where it should be?
This has also be simplified in latest commits.

Reply:
I prefer the previous div... why cannot padding sit at root level? or else set a prop and update component.. or create a common layout and read menu status from store.. but we should avoid this kind of global css as much as possible.. instead think in terms of components